<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserToChat</class>
 <widget class="QWidget" name="AddUserToChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickDialog</string>
  </property>
  <property name="windowIcon">
   <iconset theme="contact-new"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(24, 0, 36, 255), stop:0.312849 rgba(38, 8, 72, 255), stop:0.731844 rgba(44, 18, 97, 255), stop:1 rgba(50, 29, 125, 255));</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_ChatName">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 18pt &quot;Comic Sans MS&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(239, 255, 246);</string>
     </property>
     <property name="text">
      <string>ChatName(23)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Participants">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: rgb(234, 255, 252);
font: italic 14pt &quot;Comic Sans MS&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="title">
      <string>Участники</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(0, 0, 190)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_AddUser">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: rgb(234, 255, 252);
font: italic 14pt &quot;Comic Sans MS&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="title">
      <string>Добавить</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Search">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(0, 0, 255);
color: rgb(234, 255, 252);
background-color: rgb(0, 0, 70);
font: italic 14pt &quot;Comic Sans MS&quot;;</string>
        </property>
        <property name="placeholderText">
         <string>Поиск по имени</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_AddUser">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(0, 0, 190)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_AddUsers">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 24pt &quot;Comic Sans MS&quot;;
color: rgb(234, 253, 255);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(24, 0, 36, 255), stop:0.312849 rgba(38, 8, 72, 255), stop:0.731844 rgba(44, 18, 97, 255), stop:1 rgba(50, 29, 125, 255));
border: 1px solid rgb(0, 0, 190)</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
